packages/plugin-image/src/renderer.tsx

```diff
@@ -15,11 +15,13 @@ const Img = styled.img({
 export class ImageRenderer extends React.Component<ImageRendererProps> {
   public render() {
     const { state, imagePreview, disableMouseEvents } = this.props
-
+    const maxwidth = state.maxwidth.value
+    const styles = maxwidth > 0 ? { maxWidth: maxwidth + 'px' } : {}
     const image = (
       <Img
         src={imagePreview ? imagePreview.dataUrl : state.src.value}
         alt={state.description.value}
+        style={styles}
```

Comment:
see coment above - change this to `maxWidth={maxWidth}` and handle in styled component.